springbootsample/README.md

```diff
@@ -0,0 +1,21 @@
+# Run Application with SpringBoot Application
```

Comment:
minor: remove "Application with", just keep it "Run SprintBoot Application with ApplicationInsights"